@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -834,6 +834,17 @@ export class TileGeometryCreator {
                                 unitFactor *
                                 0.5;
 
+                            if ('outlineWidth' in technique && technique.outlineWidth !== undefined) {
+                              lineMaterial.outlineWidth =
+                                  getPropertyValue(
+                                    technique.outlineWidth,
+                                    mapView.zoomLevel,
+                                    mapView.pixelToWorld
+                                  ) *
+                                  unitFactor *
+                                  0.5;
```

Comment:
Why `0.5` here?
What is the difference between `Technique.outlIneWidth` and `SolidLineMaterial.outlineWidth` ?
They should be the same i guess.

(I guess this is our usual mess with semantics of `width` vs `extrusion ratio`, but at least in new code, we can have it done well.